podcasts/AppDelegate+Defaults.swift

```diff
@@ -16,15 +16,15 @@ extension AppDelegate {
 
             Settings.setShouldDeleteWhenPlayed(true)
             Settings.setHomeFolderSortOrder(order: .dateAddedNewestToOldest)
-            Settings.setMobileDataAllowed(true)
+            Settings.setMobileDataAllowed(true, userInitiated: false)
```

Comment:
What do you think about doing the inverse?

The default value of `userInitiated` is `false`. If it's user-initiated this needs to be explicit. I think it might prevent calling these functions and triggering the tracks by mistake.